build/sitecore-packages.json

```diff
@@ -287,6 +299,14 @@
         "url": "https://sitecoredev.azureedge.net/~/media/76207D15A2A540C5A4772E33D6EC0E6D.ashx",
         "hash": "577B41671B9195EE0F83E858102A64BD7F2B26E3D852D1E3669DEC609F75EB2D"
     },
+    "Sitecore JavaScript Services Server for Sitecore 10.1.0 XM 15.0.0 rev. 201020.scwdp.zip": {
```

Comment:
This should be using Sitecore Headless Rendering for v10.1 https://dev.sitecore.net/Downloads/Sitecore_Headless_Rendering/16x/Sitecore_Headless_Rendering_1600.aspx

Reply:
This is no longer required since official assets are now provided by Sitecore. I have removed this.